packages/venia-ui/lib/components/clickable.css

```diff
@@ -6,5 +6,4 @@
     line-height: 1;
     pointer-events: auto;
     text-align: center;
-    white-space: nowrap;
```

Comment:
This might be a bit too aggressive as `clickable` is the base dependency upon which the [Button](https://github.com/magento/pwa-studio/blob/develop/packages/venia-ui/lib/components/Button/button.css#L2) class composes itself. If the team is good with it, I would just reset the `white-space` rule on buttons.

I think @jimbo would have an opinion here and I remember @supernova-at doing work in this area, specifically around wrapping text in dialogs. I think in Andy's case he was avoiding a situation where you have two buttons but one wraps which presents an odd UX.

Reply:
Yes exactly.

The direction from UX is that we don't want text in buttons to be multi-line. If a client / merchant / partner has a need for it (translated text is too long, for example) then yes, they should treat that as an exception and either use a different prompt or apply a class override to wrap that specific button's text.

I agree that it is too aggressive here, and likely too aggressive in the button component as well.

Reply:
You two are correct. Not only do we not necessarily want `white-space: nowrap` on every clickable element, we don't want it on _any_ clickable element.

In early designs, we attempted to define a rule prohibiting clickable elements from wrapping. It didn't work. In some languages, authors simply can't create a meaningful call to action when they only have one line to work with; the right call to action is simply too verbose to fit, so it must wrap.